### ISSUE TITLE
It seems training not working(rewards don't converge at all)

### ISSUE THREAD

**Issue by zhiyiZeng** [MehranTaghian/DQN-Trading#9]
This repo is pretty awesome. I'm trying to run a basic demo, but the training process seems not working at all (rewards don't converge at all). However, the agent still outperforms B&H a lot.(even when the reward is negative! ) I'm confused by this situation. Is there an explanation about this?

The graph is rewards with training epochs=50.
![image](https://user-images.githubusercontent.com/20840761/186827018-4e0ba0fb-446a-4aee-8cee-7d3c4adff504.png)

**Comment by zhiyiZeng** [comment]
I tries to make learning rate smaller, but it seems that the performance gets worse. Therefore, I don't think learning rate is the issue here.

**Comment by zhiyiZeng** [comment]
I think the model learns something, but that something doesn't reflect by the rewards. I wonder why is that.

**Comment by rezapaki1376** [comment]
It is better to use a smaller self.EPS_DECAY value in BaseTrain.py, current value is 500, which for a dataset with 2-3k datapoints, is very high, and the agent takes many random actions in each episode. So, you see in training, no convergence. I reduced it to 50 and it seems to produce more stable results. Hope this helps.
